packages/beacon-node/src/network/peers/discover.ts

```diff
@@ -220,6 +220,10 @@ export class PeerDiscovery {
    * Request to find peers. First, looked at cached peers in peerStore
    */
   private async runFindRandomNodeQuery(): Promise<void> {
+    if (this.discv5.kadValues().length === 0) {
+      this.logger.error("discv5 has no kad value");
+    }
```

Comment:
This error could get very noisy, so I'm thinking how often this should log. Questions:
- If the kad table has some value at the start, can it get empty after time? I think no
- Then, the only way for kad table to be empty is there are no bootnodes

So, options:
- If the above is true, just logging if there are 0 bootnodes at the start should be equivalent?
- Else, log only on the first run with a `has_warned_empty_kad` state variable
- Or if the kad can become empty, then log once only per time interval the kad became empty

Reply:
Update: log error if there is no boot enr